learn-responsive-web-design-by-building-a-piano/index06.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Responsive Piano</title>
+</head>
+
+<body>
+
+    <div id="piano">
+        <div class="keys">
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+        </div>
+    </div>
+ 
+</body>
+</html>
+
+<!-- 
+    Now, you need to duplicate the set of 7 keys that we
```

Comment:
Break this down into two steps. You could instruct students just to copy and paste the key divs once, then to copy and paste them again in the next lesson for a total of 21 keys. No need to explain how the first set is wrapped in a `<div>`.